package.json

```diff
@@ -0,0 +1,130 @@
+{
+  "name": "@integreat-app/sticky-headroom",
+  "version": "1.0.2",
+  "engines": {
+    "node": ">=6.9",
+    "npm": ">=3.8"
+  },
+  "license": "MIT",
+  "description": "A react component for hiding the header when scrolling.",
+  "author": "Michael Markl <marklmichael98@gmail.com>",
+  "repository": "https://github.com/integreat/sticky-headroom",
+  "files": [
+    "index.js",
+    "index.js.flow"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "keywords": [
+    "headroom",
+    "sticky",
+    "react",
+    "hide",
+    "header",
+    "styled-components"
+  ],
+  "module": "index.js",
+  "scripts": {
+    "build": "node tools/build.js",
+    "test": "jest --config jest.config.json",
+    "test:coverage": "jest --config jest.config.json --coverage",
+    "test:watch": "jest --config jest.config.json --watchAll",
+    "test:update": "jest --config jest.config.json -u",
+    "lint": "yarn run eslint && yarn run stylelint",
+    "lint:fix": "yarn run eslint --fix && stylefmt -r \"src/**/*.css\"",
+    "eslint": "eslint .",
+    "stylelint": "stylelint --config stylelint.config.js \"src/**/*.{js,css}\"",
+    "flow:check-now": "flow check",
+    "flow:install-types": "flow-typed install"
+  },
+  "peerDependencies": {
+    "react": "^16.x.x",
+    "styled-components": "^3.x.x"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.0.0",
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-proposal-do-expressions": "^7.0.0",
+    "@babel/plugin-proposal-export-default-from": "^7.0.0",
+    "@babel/plugin-proposal-export-namespace-from": "^7.0.0",
+    "@babel/plugin-proposal-json-strings": "^7.0.0",
+    "@babel/plugin-proposal-logical-assignment-operators": "^7.0.0",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.0.0",
+    "@babel/plugin-proposal-numeric-separator": "^7.0.0",
+    "@babel/plugin-proposal-optional-chaining": "^7.0.0",
+    "@babel/plugin-proposal-pipeline-operator": "^7.0.0",
+    "@babel/plugin-proposal-throw-expressions": "^7.0.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
+    "@babel/plugin-syntax-import-meta": "^7.0.0",
+    "@babel/plugin-transform-react-jsx": "^7.0.0",
+    "@babel/plugin-transform-runtime": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-flow": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/runtime": "^7.0.0",
+    "@integreat-app/eslint-config-babel": "^0.3.1",
+    "@integreat-app/eslint-config-flow": "^0.3.2",
+    "@integreat-app/eslint-config-jest": "^0.2.1",
+    "@integreat-app/eslint-config-node": "^0.3.2",
+    "@integreat-app/eslint-config-react": "^0.3.3",
+    "assets-webpack-plugin": "^3.9.6",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-eslint": "^9.0.0",
+    "babel-jest": "^23.6.0",
+    "babel-loader": "^8.0.0",
+    "babel-plugin-styled-components": "^1.5.1",
+    "browser-sync": "^2.24.7",
```

Comment:
Should not be needed. also look though the other devDependencies

Reply:
Hmm... yeah I wanted to do that. The reason I didn't do it, was because I wanted to build some demo page which would introduce the webpack-stack again...

Reply:
Hm ok, I would still keep it clean until we need it. Probably won't need browser-sync :)

Reply:
Will leave them here for now.